instrumentation/opentelemetry_tesla/lib/middleware/opentelemetry_tesla_middleware.ex

```diff
@@ -0,0 +1,93 @@
+defmodule Tesla.Middleware.OpenTelemetry do
```

Comment:
Is using `Tesla.` as the module prefix, "namespace", the pattern used for Tesla middlewares that are from third parties?

Reply:
Yes, it is